pkg/config/expand.go

```diff
@@ -186,6 +187,33 @@ func (bp Blueprint) expandBackend(grp *Group) {
 	}
 }
 
+func kubectlProviderRequiredModule(grp *Group) (bool, Module) {
+	for _, mod := range grp.Modules {
+		if strings.Contains(mod.Source, "gke-cluster") || strings.Contains(mod.Source, "pre-existing-gke-cluster") {
```

Comment:
`strings.Contains(mod.Source, "gke-cluster")` covers both "gke-cluster" and  "pre-existing-gke-cluster".

I would suggest to check full module path ("modules/scheduler/...") to prevent any name similarities later. ex. `gke-cluster-something`

Reply:
Thanks for pointing this out, change pushed.

Reply:
There is existing code that solves roughly this problem here: 894520281b60c0b93e856d0df5d2e1d9f20bdd16